src/profile-logic/transforms.js

```diff
@@ -318,13 +318,24 @@ export function stringifyTransforms(
     .join(';');
 }
 
-export function getTransformLabels(
+export type TransformLabeL10nIds = {|
+  +l10nId: string,
+  +item: string,
+|};
+
+/**
+ * Gets all applied transforms and returns their labels as l10n Ids with the
+ * items associated to them. The `item`s can be a resource, function, or thread
+ * name. They are being passed in the `Localized` component to create the
+ * transform strings as desired.
+ */
+export function getTransformLabelL10nIds(
```

Comment:
The implementation in this function is a bit different to what we have normally. This was done to make it work from the `profile-logic` (without the redux state). All the l10nIds include also an item that we pass inside the strings as variables.